openmm_md/plain_md.ipynb

```diff
@@ -0,0 +1,369 @@
+{
```

Comment:
<p>1)</p><p> </p><p>The bullet points go from 8 to 10 (also there's a weird indentation for 10).</p><p> </p><p>If you felt like it, it might be worth seeing if a table format might be cleaner here (but that's really just if you want to play around with formatting).</p><p> </p><p>2)</p><p> </p><p>"not the entire system with all the atoms should be saved" - a bit nitpicky, but this is a bit difficult to read because of the "not". Maybe "Special output indices can be defined to select which portions of the system should be saved" (it's the same but without the "not"). We should also make it clear as to what the default is (I think it's "not water"?)</p><p> </p><p>It would be good to highlight the specific attributes that would need changing if possible. e.g. <code>output_indices</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/97/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>That's a good idea! I moved the bullet points into a table, changed the sentence to your suggestion and changed some of the descriptions to highlight the code attributes. I hope this is intuitive for the user.</p>